proposed/http-message.md

```diff
@@ -85,7 +85,7 @@ convenience of working with headers as strings, headers can be retrieved from
 an instance of a ``MessageInterface`` as an array or string. Use the
 `getHeader()` method to retrieve a header value as a string containing all
 header values of a header by name concatenated with a comma.
-Use `getHeaderAsArray()` to retrieve an array of all the header values for a
+Use `getHeaderLines()` to retrieve an array of all the header values for a
```

Comment:
The signature is `getHeaderLines($headerName)`; it should raise an exception if no header name is presented.

Essentially, I'm not sure there's a good name here. We're talking about getting plural lines/values for a single header, versus a set of headers. There is no name that will be completely disambiguous unless we choose to return to using hungarian notation, which also sucks.